router/broker.go

```diff
@@ -117,28 +115,13 @@ func (b *broker) RemoveSession(sub *Session) {
 
 // Close stops the broker and waits message processing to stop.
 func (b *broker) Close() {
-	if b.Closed() {
-		return
-	}
 	close(b.reqChan)
```

Comment:
Thought I guarantees they would all be done by the time this was closed.  Will re-evaluate.